### ISSUE TITLE
Better documentation for contributing / building etc

### ISSUE THREAD

**Issue by arianvp** [arianvp/dhall-kubernetes#71]
@jvanbruegge noted they struggled to get things to build. I should spend more time explaining how it works and make sure it isn't actually broken

**Comment by jvanbruegge** [comment]
Yeah, and as said, I am using Arch which does not really work well with nix. Having a docker container that I could just mount the project folder into that has all the nix stuff included would allow me too to build and test everything
